src/metabase/driver.clj

```diff
@@ -149,6 +149,12 @@
      *  `:set-timezone`
      *  `:standard-deviation-aggregations`")
 
+  (field-avg-length ^Integer [this, ^FieldInstance field]
+    "*OPTIONAL*. Return the avergage length of FIELD. Default implementation does this in Clojure-land.")
+
+  (field-percent-urls ^Double [this, ^FieldInstance field]
```

Comment:
I'd like to have a chance to talk with you about the need to reintroduce these functions on the main driver interface.  I specifically pulled these out when I did the syncing refactor because I'm not convinced this is a great pattern to promote in the sync code.

So lets grab a little time tomorrow and we can talk about it.